helm/alfresco-content-services/templates/deployment-filestore.yaml

```diff
@@ -83,20 +83,6 @@ spec:
           - name: data
             mountPath: {{ .Values.persistence.filestore.data.mountPath }}
             subPath: {{ .Values.persistence.filestore.data.subPath }}
-      {{- if and .Values.persistence.filestore.enabled .Values.persistence.chownWithDynamicProvisioning }}
```

Comment:
chownWithDynamicProvisioning needs to be removed from values.yaml since it is now unused